### DISCUSSION TITLE
更新了之后只能手动刮取

### DISCUSSION THREAD

**Discussion by gtise** [metatube-community/jellyfin-plugin-metatube#103] [Q&A]
![image](https://user-images.githubusercontent.com/61878004/208248485-d5859eaf-a172-4f3f-ad7d-0204453a82ef.png)
这三个选项我都尝试了，但是没有成功。
手动是可以使用的，也 可以保存下来。
![image](https://user-images.githubusercontent.com/61878004/208248507-4ea7613b-bb95-47b8-babd-5f9b49f396bd.png)
这就有点搞不懂了。请求帮助

**Comment by xjasonlyu** [comment]
Organize Metadata 计划任务运行一下，会整理一遍迁移的数据就好了

**Comment by kitkw** [comment]
我在新版也不能自动搜刮了....
以前影片放到目录路径，就会实时监控 自动搜刮的。现在都要手动触发......
